### ISSUE TITLE
v2-v3 migration?

### ISSUE THREAD

**Issue by CourtHive** [danalloway/react-country-flag#47]
https://danalloway.github.io/react-country-flag appears to be broken
attempted upgrade but no idea what the breaking changes are!

**Comment by Amorem** [comment]
Unable to upgrade, got this message: 

"Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports."

**Comment by danalloway** [comment]
@Amorem how are you using the component? please post your code so I can see what is going on, thanks!

**Comment by danalloway** [comment]
@CourtHive no migration, only thing that has changed are Typescript types
yea, the demo is busted, will fix that at some one, sorry!

**Comment by danalloway** [comment]
@CourtHive added a note to the Readme, thanks for pointing that out

**Comment by danalloway** [comment]
@Amorem please open a new issue with code from your project showing how you're using the component and we'll diagnose there, thanks.

**Comment by CourtHive** [comment]
Dan,

```js
git clone git@github.com:CourtHive/tods-score-grid.git

yarn install
yarn storybook
```

All is well

Edit `package.json` and bump the version of `react-country-flag`

```js
yarn install 
yarn storybook
```

"Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports."

**Comment by danalloway** [comment]
Thanks, but I need  to see how you’re using the component.

**Comment by CourtHive** [comment]
```
      <ReactCountryFlag
        alt={alt}
        countryCode={nationalityCode}
        loading="lazy"
        style={{
          height: '100%',
          width: '100%',
          verticalAlign: 'initial'
        }}
        svg
      />
```

**Comment by CourtHive** [comment]
I think the problem is that you no longer have a default export

```
import ReactCountryFlag from "react-country-flag"
```

no longer works, even though that is what you give in your example

```
import { ReactCountryFlag } from "react-country-flag"
```

does work... and this is what you have in `blah.text.tsx`

**Comment by Amorem** [comment]
Hello Dan, 

Env: NextJS 12

Code:
<ReactCountryFlag
            className="mr-4"
            countryCode={course.locale.slice(3, 5)}
            style={{
              fontSize: "20px",
            }}
            aria-label={course.locale}
 />

but also try with examples from the readme (<ReactCountryFlag countryCode='US'/>) and got same error.

**Comment by Amorem** [comment]
> I think the problem is that you no longer have a default export
> 
> ```
> import ReactCountryFlag from "react-country-flag"
> ```
> 
> no longer works, even though that is what you give in your example
> 
> ```
> import { ReactCountryFlag } from "react-country-flag"
> ```
> 
> does work... and this is what you have in `blah.text.tsx`

Solved !

**Comment by danalloway** [comment]
fixed in v3.0.2
also fixed the demo page
closes #48
